tools/run-task.sh

```diff
@@ -11,16 +11,19 @@ cluster="task_runner"
 govuk_env="test"
 
 function show_help {
-  echo "Usage: cd govuk-infrastructure && gds aws govuk-tools-internal-admin ./tools/run-rake-task -a frontend rake db:migrate"
+  echo "Usage: cd govuk-infrastructure && gds aws govuk-test-poweruser ./tools/run-rake-task -a frontend -v live rake db:migrate"
```

Comment:
Actually this needs to be `-admin` because there's an IAM policy that needs attaching.